Homework_15/homework_15.py

```diff
@@ -0,0 +1,24 @@
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        self.setattr('side_a', side_a)
+        self.setattr('angle_a', angle_a)
+
+    def setattr(self, name, value):
```

Comment:
It is better to use dunder method __setattr__ for such logic implementation. Please review that moment.